bsp_diff/caas/device/intel/civ/host/vm-manager/0046-Multi-Android-VM-support.patch

```diff
@@ -0,0 +1,347 @@
+-        sed -i "/^\[rpmb\]$/,/^\[/ s#^data_dir.*#data_dir=$CIV_WORK_DIR#" $USER_HOME/.intel/.civ/civ-sriov.ini
++        sed -i "/^\[rpmb\]$/,/^\[/ s#^bin_path.*#bin_path=$CIV_WORK_DIR\/$VM_NAME\/scripts\/rpmb_dev#" $USER_HOME/.intel/.civ/civ-sriov.ini
++        sed -i "/^\[rpmb\]$/,/^\[/ s#^data_dir.*#data_dir=$CIV_WORK_DIR\/$VM_NAME#" $USER_HOME/.intel/.civ/civ-sriov.ini
+ 
+-        sed -i "/^\[aaf\]$/,/^\[/ s#^path.*#path=$CIV_WORK_DIR/scripts/aaf#" $USER_HOME/.intel/.civ/civ-sriov.ini
++        sed -i "/^\[aaf\]$/,/^\[/ s#^path.*#path=$CIV_WORK_DIR/$VM_NAME/scripts/aaf#" $USER_HOME/.intel/.civ/civ-sriov.ini
+         sed -i "/^\[aaf\]$/,/^\[/ s#^support_suspend.*#support_suspend=disable#" $USER_HOME/.intel/.civ/civ-sriov.ini
+ 
+         sed -i "/^\[passthrough\]$/,/^\[/ s/^passthrough_pci/#passthrough_pci/" $USER_HOME/.intel/.civ/civ-sriov.ini
++
++        mv $USER_HOME/.intel/.civ/civ-sriov.ini $USER_HOME/.intel/.civ/$VM_NAME.ini
+ }
+ 
+ function copy_civ_ini() {
+@@ -39,6 +48,30 @@ function copy_civ_ini() {
+         chmod 0666 $USER_HOME/.intel/.civ/civ-sriov.ini
+ }
+ 
++function read_args() {
++        while [[ $# -gt 0 ]]; do
++                case $1 in
++                        -v)
++                                VSOCK_ID=$2
++                                shift
++                                ;;
++
++                        -p)
++                                ADB_PORT=$2
++                                shift
++                                ;;
++
++                        -n)
++                                VM_NAME=$2
++                                shift
++                                ;;
++                esac
++                shift
++        done
++}
++
++read_args "$@"
++
+ copy_civ_ini
+ setup_civ_ini
+ 
+diff --git a/scripts/setup_host.sh b/scripts/setup_host.sh
+index 0c1e0a1..71cde03 100755
+--- a/scripts/setup_host.sh
++++ b/scripts/setup_host.sh
+@@ -11,6 +11,7 @@ set -eE
+ reboot_required=0
+ QEMU_REL="qemu-7.1.0"
+ skip_install_qemu=false
++skip_initial_host_setup=false
+ 
+ #Directory to keep versions of qemu which can be reused instead of downloading again
+ QEMU_CACHE_DIR="$HOME/.cache/civ/qemu"
+@@ -19,6 +20,10 @@ CIV_WORK_DIR=$(pwd)
+ CIV_GOP_DIR=$CIV_WORK_DIR/GOP_PKG
+ CIV_VERTICAl_DIR=$CIV_WORK_DIR/vertical_patches/host
+ VM_MANAGER_VERSION=v1.2.3
++VSOCK_ID=3
++VM_NAME=Android_CIV
++ADB_PORT=5555
++USER_HOME=$(getent passwd $SUDO_USER | cut -d: -f6)
+ 
+ #---------      Functions    -------------------
+ function error() {
+@@ -575,6 +580,26 @@ function parse_arg() {
+                 setup_remote_infer || return -1
+                 ;;
+ 
++	    -s)
++		echo "Skip initial setup"
++                skip_initial_host_setup=true
++		;;
++
++	    -v)
++                VSOCK_ID=$2
++		shift
++		;;
++
++	    -b)
++                ADB_PORT=$2
++		shift
++                ;;
++
++            -n)
++                VM_NAME=$2
```

Comment:
If the user do the following:
1) run setup_host.sh to create VM1
2) run setup_host_multiple_xxx.sh to create VM2
3) at a later time, run setup_host_multiple_xxx.sh to create VM3

If the user has different names for VM2 and VM3, will this create a problem?
Does it help if we do not allow customised name and have a standardised name instead?